tests/integration/test_kfp_functional.py

```diff
@@ -73,11 +73,12 @@ async def test_build_and_deploy(ops_test: OpsTest, request, lightkube_client):
         status="active",
         raise_on_blocked=False,  # These apps block while waiting for each other to deploy/relate
         raise_on_error=True,
-        timeout=1800,
+        timeout=3600,
+        idle_period=120,
     )
 
-
 # ---- KFP API Server focused test cases
+@pytest.mark.skip
```

Comment:
Are these still needed, or have the tests been fixed?

Reply:
Yes. This needs to be re-enabled.